lib/home_screen/home_screen.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:flutter/material.dart';
+import 'package:lachies_life_planner/home_screen/widgets/text_icon_button.dart';
+import 'package:lachies_life_planner/shared/size_config.dart';
+
+class HomeScreen extends StatelessWidget {
+  HomeScreen({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    SizeConfig().init(context);
```

Comment:
What happens when we don't load from the Home Screen?  This should be done from `main.dart`

Reply:
No, because this must be called AFTER `MaterialApp` is initialised.